src/hotspot/share/runtime/globals.hpp

```diff
@@ -2026,8 +2026,11 @@ const intx ObjectAlignmentInBytes = 8;
              "Mark all threads after a safepoint, and clear on a modify "   \
              "fence. Add cleanliness checks.")                              \
                                                                             \
-  develop(bool, TraceOptimizedUpcallStubs, false,                              \
-                "Trace optimized upcall stub generation")                      \
+  develop(bool, TraceOptimizedUpcallStubs, false,                           \
+                "Trace optimized upcall stub generation")                   \
+  product(bool, UseSharedStubs, false, DIAGNOSTIC,                          \
```

Comment:
Please don't add a new option. We already have a plenty of them.

Reply:
I have removed it.